SwiftUiMovieDb/Views/MoviePosterCard.swift

```diff
@@ -11,31 +11,33 @@ import Kingfisher
 struct MoviePosterCard: View {
     
     let movie: Movie
-    let cache = ImageCache.default
+    private let imageCornerRadius: CGFloat = 8
+    private let imageShadowRadius: CGFloat = 4
+    private let frameWidth: CGFloat = 204
```

Comment:
Let's try to use relative height to screen width for such big dimensions.

eg: frameWidth = screenWidth * 0.7 and height = screenWidth

Let's see if this works.